src/components/m-form-group/_template.js

```diff
@@ -23,7 +23,7 @@ export default function ({
         </div>
 
         ${childrenFragment}
-        <span class="m-form-group__error">${error}</span>
+        <span class="m-form-group__error">${error || '&nbsp;'}</span>
```

Comment:
@raphaellueckl can we wrap <span class="m-form-group__error"> in a IF so that if no error is there it wont render the span tag?

Reply:
I guess we have to. Then we have, what we had before. I guess we need to discuss, how we handle github issues.

Reply:
@raphaellueckl i have not red all the above arguments, but i will have a look so i have more context. Just at looking at the code seems like a better way to just remove it if is not there. If the height is required, might be easier to have a min-height or something like that so that.